corehq/apps/export/utils.py

```diff
@@ -88,6 +88,20 @@ def _convert_index_to_path(index):
         return ['data'] + index.split('.')[1:]
 
 
+def _convert_index_to_path_nodes(index):
+    from corehq.apps.export.models.new import MAIN_TABLE
+    from corehq.apps.export.models.new import PathNode
+    if index == '#':
+        return MAIN_TABLE
+    elif _is_repeat(index):
+        return [PathNode(name='data')] + [
+            PathNode(name=n, is_repeat=True)
+            for n in _strip_repeat_index(index).split('.')[1:]
+        ]
```

Comment:
@benrudolph This function assumes that all elements in the path (except for "data") are repeats. That might be false. Not sure if I understand the semantics the old index strings well enough to say without some more experimenting, but basically we need to be able to distinguish which elements of the path are repeats, and which are not.